.scalafmt.conf

```diff
@@ -1,5 +1,5 @@
-version = "2.0.1"
-maxColumn = 120
+version = "2.5.0-RC2"
+maxColumn = 160
 align = some
```

Comment:
As we make changes in formatter, can we make him work only if we use format command in IDE - not all the time?

Reply:
've discussed
it's about IDEA